carbon-design-system/carbon
# fix(toggle): remove presentational checkmark svg from accessibility tree

Closes #17676 

[Reference](https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html#:~:text=SVGs%20that%20are%20decorative)

#### Changelog

**Changed**

- add aria-hidden and focusable:false to checkmark svg

#### Testing / Reviewing

1. Open Toggle component description on storybook
2. Go to playground and choose the "sm" value of "size" prop
3. Open the result in fullscreen/iframe
4. Scan it with IBM Equal Access Accessibility Checker
5. There should be no violation regarding the svg having no accessible name
